packages/frontend/src/components/dialogs/ConfirmDeleteMessage.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+
+import Dialog, {
+  DialogBody,
+  DialogContent,
+  DialogFooter,
+  FooterActions,
+} from '../Dialog'
+import FooterActionButton from '../Dialog/FooterActionButton'
+import useTranslationFunction from '../../hooks/useTranslationFunction'
+
+import type { DialogProps } from '../../contexts/DialogContext'
+import { BackendRemote, Type } from '../../backend-com'
+import { C } from '@deltachat/jsonrpc-client'
+
+export type Props = {
+  accountId: number
+  msg: Type.Message
+  canSend: boolean
+} & DialogProps
+
+export default function ConfirmDeleteMessageDialog(props: Props) {
+  const tx = useTranslationFunction()
+
+  const { accountId, msg, onClose } = props
+
+  const deleteForAllPossible =
```

Comment:
I noticed that this is also available in "saved messages". I think it doesn't make sense there.

Android checks for this https://github.com/deltachat/deltachat-android/pull/3633/files#diff-6046f8bf3371c6d149fcd32fc2b3ac0e1536a66e34b13e0096208086136529a5R360

Reply:
Added filter in with 585369ff27324